package.json

```diff
@@ -16,6 +16,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
+    "pretest": "sh setup.sh",
```

Comment:
One point of note: The .travis.yml already does this. If you're doing it here, you need to remove it from [here](https://github.com/strongloop/loopback-connector-cloudant/blob/master/.travis.yml)